docs/supported_apis/dataframe_supported.rst

```diff
@@ -83,13 +83,19 @@ default to pandas.
 +----------------------------+---------------------------+------------------------+----------------------------------------------------+
 | ``copy``                   | `copy`_                   | Y                      |                                                    |
 +----------------------------+---------------------------+------------------------+----------------------------------------------------+
-| ``corr``                   | `corr`_                   | Y                      |                                                    |
+| ``corr``                   | `corr`_                   | Y                      | Correlation floating point precision may slightly  |
+|                            |                           |                        | differ from pandas. For now pearson method is      |
+|                            |                           |                        | available only.                                    |
+|                            |                           |                        | For other methods defaults to pandas.              |
 +----------------------------+---------------------------+------------------------+----------------------------------------------------+
 | ``corrwith``               | `corrwith`_               | D                      |                                                    |
 +----------------------------+---------------------------+------------------------+----------------------------------------------------+
 | ``count``                  | `count`_                  | Y                      |                                                    |
 +----------------------------+---------------------------+------------------------+----------------------------------------------------+
-| ``cov``                    | `cov`_                    | Y                      |                                                    |
+| ``cov``                    | `cov`_                    | Y                      | Covariance floating point precision may slightly   |
```

Comment:
We recently merged `cov` implementation considering NA/null values into master and forgot to update the doc. So, the second sentence should be deleted (I mean this: `If DataFrame contains at least one NA/null value, then defaults to pandas.`)